src/Resources/translations/SonataAdminBundle.en.xliff

```diff
@@ -9,6 +9,14 @@
                 <source>sonata_media</source>
                 <target>Media Library</target>
             </trans-unit>
+            <trans-unit id="form.label_icon">
+                <source>form_group.gallery</source>
+                <target>form_group.gallery</target>
+            </trans-unit>
+            <trans-unit id="form.label_icon">
+                <source>form_group.options</source>
+                <target>form_group.options</target>
```

Comment:
Options

Reply:
Not touching this file, I reverted the changes, this is for SonataAdminBundle translation domain (we should probably move this to the SonataMediaBundle domain, but that will belong to another Pr.